runbotics/runbotics-orchestrator-ui/src/src-app/hooks/useOwner.ts

```diff
@@ -0,0 +1,11 @@
+import useAuth from './useAuth';
+
+export const useOwner = () => {
+    const { user } = useAuth();
+
+    return (ownerId: number) => {
+        const isOwner = user.id === ownerId;
+
+        return isOwner;
+    };
```

Comment:
imo the hook should have:

const isOwner (ownerId: number) => user.id === ownerId;
return ownerId

In case we want to extend this in the future and imo it increases readability